exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/okhttp/OkHttpExporterBuilder.java

```diff
@@ -137,6 +147,21 @@ public OkHttpExporter<T> build() {
       clientBuilder.addInterceptor(new RetryInterceptor(retryPolicy, OkHttpExporter::isRetryable));
     }
 
+    if (authenticator != null) {
+      Authenticator finalAuthenticator = authenticator;
+      // Generate and attach OkHttp Authenticator implementation
+      clientBuilder.authenticator(
+          (Route route, Response rspns) -> {
+            Request.Builder requestBuilder = rspns.request().newBuilder();
+            finalAuthenticator.getHeaders(
+                ah -> {
```

Comment:
I don't know this API. What's the type of `ah`? Can we make the variable name more expressive? Thanks!

Reply:
expanded to authHeaders